suprb/optimizer/rule/mutation.py

```diff
@@ -69,47 +87,108 @@ class Normal(RuleMutation):
     def ordered_bound(self, rule: Rule, random_state: RandomState):
         # code inspection gives you a warning here but it is ineffectual
         rule.match.bounds += random_state.normal(scale=self.sigma,
-                                        size=rule.match.bounds.shape)
+                                                 size=rule.match.bounds.shape)
         rule.match.bounds = np.sort(rule.match.bounds, axis=1)
 
+    def unordered_bound(self, rule: Rule, random_state: RandomState):
+        rule.match.bounds += random_state.normal(scale=self.sigma,
+                                                 size=rule.match.bounds.shape)
+
+    def centre_spread(self, rule: Rule, random_state: RandomState):
+        assert isinstance(self.sigma, np.ndarray) and self.sigma.shape[0] == 2
+        bounds = rule.match.bounds
+        bounds[:, 0] += random_state.normal(scale=self.sigma[0],
+                                            size=rule.match.bounds.shape[0])
+        bounds[:, 1] += random_state.normal(scale=self.sigma[1],
+                                            size=rule.match.bounds.shape[0])
```

Comment:
It might be worth considering writing a helper function here, since the two lines only differ in the index. Perhaps a loop from 0 to 1 might also be worth considering